flink-libraries/flink-table/src/main/scala/org/apache/flink/table/runtime/aggregate/AggregateUtil.scala

```diff
@@ -785,7 +785,7 @@ object AggregateUtil {
     (propPos._1, propPos._2)
   }
 
-  private def transformToAggregateFunctions(
```

Comment:
@fhueske I could do that - just let me know if this is what you want. For me it would definitely make sense